cmd/manager/main.go

```diff
@@ -69,7 +72,7 @@ var (
 	defaultSystemNamespace = "olmv1-system"
 )
 
-const authFilePath = "/etc/operator-controller/auth.json"
+const authFilePath = "/tmp/operator-controller/auth.json"
```

Comment:
This change is interesting. I think I understand the motivation behind it, but I feel like this is something we should chat through.

Would it make sense to use a path directly in the current working directory, and then tell the controller and the unpacker the absolute path for it? Maybe something like `$PWD/operator-controller-global-pull-secret.json` (but maybe $TMPDIR is fine too; we should use `$TMPDIR`, not `/tmp` if we stick with an absolute location).

IMO, we should:
- do it in a place where we don't have to create a directory to store it.
- delete it when the process is shutting down.

Reply:
Fyi https://docs.google.com/document/d/1BXD6kj5zXHcGiqvJOikU2xs8kV26TPnzEKp6n7TKD4M/edit?disco=AAABTsk4FzQ 

* We run into permission issues if we try to create the file in /etc. 
* $PWD is root (/), so we run into permission issues trying go create /operator-controller-global-pull-secret.json too. 
* I don't see $TMPDIR set when I deploy locally. We could do something like 
```
tmpDir := os.Getenv("TMPDIR")
if tmpDir == "" {
    tmpDir = "/tmp"
} 
```

but is that really necessary?

Reply:
Use `os.TempDir()`. It's basically that.

Reply:
Done in a [new commit](https://github.com/operator-framework/operator-controller/pull/1322/commits/527e324664d0761aa4cd6b793b2e8f906d652604)

Reply:
Looks good!